pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -341,25 +387,24 @@ func (b *OFBridge) AddFlowsInBundle(addflows []Flow, modFlows []Flow, delFlows [
 		return nil
 	}
 
-	// Install new Openflow entries with the opened bundle.
-	if err := syncFlows(addflows, openflow13.FC_ADD); err != nil {
-		return err
-	}
-	// Modify existing Openflow entries with the opened bundle.
-	if err := syncFlows(modFlows, openflow13.FC_MODIFY_STRICT); err != nil {
-		return err
-	}
-	// Delete Openflow entries with the opened bundle.
-	if err := syncFlows(delFlows, openflow13.FC_DELETE_STRICT); err != nil {
-		return err
+	// Add Group modification messages in advance of Flow modification messages. As the bundle is adding messages in
```

Comment:
Consider:
"Flow modification messages, so it can ensure the dependent Group exists when adding a new Flow entry.

Reply:
Sure, updated.